buf/internal/lint.bzl

```diff
@@ -23,7 +23,7 @@ _DOC = """
 For more info please refer to the [`buf_lint_test` section](https://docs.buf.build/build-systems/bazel#buf-lint-test) of the docs.
 """
 
-_TOOLCHAIN = str(Label("//tools/protoc-gen-buf-lint:toolchain_type"))
+_TOOLCHAIN = "@rules_buf//buf:toolchain_type"
```

Comment:
Won't using `@rules_buf` fail when the imported repository is not named that?

Reply:
the rule repo name should be consistent, it's part of your API essentially...